dp/cloud/python/magma/radio_controller/services/dp/service.py

```diff
@@ -98,15 +105,19 @@ def CBSDRegister(self, request: CBSDRequest, context) -> CBSDStateResult:
         """
 
         logger.info(f"Registering CBSD {request.serial_number=}")
+        request_type = result_type = 'CBSDRegister'  # noqa: WPS429
```

Comment:
Do we actually need 2 variables for that? One should be enough, especially since they are not changed anywhere, unless I'm missing something

Reply:
i only did it for readibility. Will make it into one var called message_type